CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
-# 0.18.5 (wip)
+# 1.0.0 (2023-10-02)
 
 ## Release
 
-- nns `v0.16.8`
-- sns `v0.0.23`
-- cmc `v0.0.19`
-- ledger `v0.0.16`
-- ckBTC `v0.0.12`
-- ic-management `v0.0.9`
-- utils `v0.0.23`
-- nns-proto `v0.0.9`
+- nns `v1.0.0`
+- sns `v1.0.0`
+- cmc `v1.0.0`
+- ledger `v1.0.0`
+- ckBTC `v1.0.0`
+- ic-management `v1.0.0`
+- utils `v1.0.0`
+- nns-proto `v1.0.0`
```

Comment:
Don't we want to wait until ledger-icp for it?

Maybe wait until the LedgerCanister is not in nns.

Reply:
No on the contrary, that's exactly the idea!

Release v1 and then v2 === breaking changes when we introduce ledger-icp/ledger-icrc

Reply:
Discussed offline.